### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by mxcl** [teaxyz/teaBASE#6] [Announcements]
## What's Changed
* Universal builds, Intel Macs now supported
* Add `~/.zshenv` to dotfile sync
* Allow and encourage editing the git identity (user/email config)
* ensure we get semver by @jhheider in https://github.com/teaxyz/teaBASE/pull/2

**Full Changelog**: https://github.com/teaxyz/teaBASE/compare/v0.2.0...v0.3.0

**Comment by NamasayaA905** [comment]
what ????

**Comment by Humsad** [comment]
Nice one
